assets/ca/ca.querytranslator.js

```diff
@@ -292,27 +258,29 @@ var caUI = caUI || {};
 	 * Use the given `queryValue` to assign a `value` and `condition` to the given `rule`.
 	 * @param {Object} rule
 	 * @param {String} queryValue
-	 * @parma {Boolean} negation
+	 * @param {Boolean} negation
+	 * @param {String} wildcardPrefix
+	 * @param {String} wildcardSuffix
 	 */
-	assignOperatorAndValue = function (rule, queryValue, negation) {
-		var i, j, mapping, matches;
-		// Find the most specific matching regular expression, which gives the operator, and the regular expression for
-		// parsing the value into a scalar or array value, depending on the number of capturing groups.
-		for (i = 0; i < REGEX_OPERATOR_MAP.length; ++i) {
-			mapping = REGEX_OPERATOR_MAP[i];
-			matches = mapping.regex.exec(queryValue);
-			if (matches) {
-				rule.operator = negation ? mapping.negatedOperator : mapping.operator;
-				if (matches.length < 2) {
-					rule.value = undefined;
-				} else if (matches.length === 2) {
-					rule.value = matches[1];
-				} else {
-					rule.value = [];
-					for (j = 1; j < matches.length; ++j) {
-						rule.value.push(matches[j]);
-					}
-				}
+	assignOperatorAndValue = function (rule, queryValue, negation, wildcardPrefix, wildcardSuffix) {
```

Comment:
This method was rewritten, don't try to compare the two, they don't relate to each other apart from setting the same properties on `rule`.